apps/linode-marketplace-influxdb/provision.yml

```
@@ -0,0 +1,21 @@
+# linode ansible playbook
+
+# linodes
+- name: setting up environment 
+  hosts: localhost
+  connection: local
+  any_errors_fatal: true
+  user: root
+  vars_files:
+    - group_vars/linode/vars
+  tasks:
+    - name: generating sudo user and password 
+      import_tasks: ../linode_helpers/roles/sudouser/tasks/main.yml
+
+    - name: generate admin_password
+      lineinfile: 
+        insertafter: EOF
+        path: group_vars/linode/vars
+        line: |
+          admin_password: {{ lookup('password', '/dev/null length=45 characters=ascii_letters,digits') }}
```

```suggestion
          admin_password: {{ lookup('password', '/dev/null length=45 chars=ascii_letters,digits') }}
```

This task fatals out. We can't see the output because `no_log` is set to true:
```output
Error was a <class 'ansible.errors.AnsibleError'>, original message: Unrecognized parameter(s) given to password lookup: characters. Unrecognized parameter(s) given to password lookup: characters
```

---

Corrected in [b62ab04](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/b62ab040728a1c112d0f5894c307bfd2f1dd3f3c).